### PR TITLE
add Task1 assignment

### PR DESCRIPTION
**Pull request by Dhairaya02**

### CODE REVIEW

File: Open Source/Git1/Ashutosh-Khanduala.txt [hunk 1]
```diff
@@ -1,4 +0,0 @@
-Ashutosh Khanduala
-ETC A 2nd Yr
-1902070030
-I love Open source
```
Comment (ashuvssut) [inline Open Source/Git1/Ashutosh-Khanduala.txt:4]: Why did you delete this file?

Comment (Dhairaya02) [reply Open Source/Git1/Ashutosh-Khanduala.txt:4]: Didn't know had to keep the original one.

Comment (ashuvssut) [reply Open Source/Git1/Ashutosh-Khanduala.txt:4]: it is clearly told in task to "create a new file"

So dont worry...  create a new file and restore Ashutosh-khanduala.txt

Comment (ashuvssut) [reply Open Source/Git1/Ashutosh-Khanduala.txt:4]: push again

This PR will be updated automatically after you push it

Comment (Dhairaya02) [reply Open Source/Git1/Ashutosh-Khanduala.txt:4]: Done

**Comment by ashuvssut** [comment]
```
git reset --soft HEAD~2
git restore --staged Open\ Source/Git1/Ashutosh-Khanduala.txt
git restore Open\ Source/Git1/Ashutosh-Khanduala.txt
```
now make a commit again "task1 completed"
push it

File: Open Source/Git1/Ashutosh-Khanduala.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
 Ashutosh Khanduala
 ETC A 2nd Yr
 1902070030
-I love Open source
```
Comment (ashuvssut) [inline Open Source/Git1/Ashutosh-Khanduala.txt:4]: I dont know why this line shows like its deleted.

But you can follow those commands to resolve

Comment (ashuvssut) [reply Open Source/Git1/Ashutosh-Khanduala.txt:4]: do

```
git reset --soft HEAD~2
git restore --staged Open\ Source/Git1/Ashutosh-Khanduala.txt
git restore Open\ Source/Git1/Ashutosh-Khanduala.txt
```
now make a commit again "task1 completed"
push it

Comment (ashuvssut) [reply Open Source/Git1/Ashutosh-Khanduala.txt:4]: Ah I forgot...

to push do `git push -f origin main`

**Comment by ashuvssut** [comment]
close this PR.

duplicate of #8
